src/components/common/account/favorite-team-header.tsx

```diff
@@ -0,0 +1,119 @@
+'use client';
+
+import useIsDesktop from '@/lib/hooks/useIsDesktop';
+import useIsMobile from '@/lib/hooks/useIsMobile';
+import clsx from 'clsx';
+import Image from 'next/image';
+import { useEffect, useState } from 'react';
+
+export default function FavoriteTeamHeader({
+	isSignup,
+	teamCount,
+	onClickEditButton,
+}: {
+	isSignup: boolean;
+	teamCount: number;
+	onClickEditButton: () => void;
+}) {
+	// 프로필 설정 페이지에서 우측 편집 버튼
+	const [isEditButtonVisible, setIsEditButtonVisible] = useState(!isSignup);
+
+	const handleEditButtonClick = () => {
+		setIsEditButtonVisible(!isEditButtonVisible);
+		onClickEditButton();
+	};
+
+	// help circle
+	const isDesktop = useIsDesktop();
+	const isMobile = useIsMobile();
+	const [isTooltipVisible, setIsTooltipVisible] = useState(false);
+
+	const handleHelpCircle = (e: React.MouseEvent) => {
```

Comment:
(이건 당장 수정 안 하셔도 될 듯 합니다)

이런 식으로 if문으로 분기하면 훨씬 간단해질 것 같아요!!

```suggestion
	const handleHelpCircle = (e: React.MouseEvent) => {
  if (isDesktop) {
    if (e.type === 'mouseover') setIsTooltipVisible(true);
    if (e.type === 'mouseleave') setIsTooltipVisible(false);
  } else {
    if (e.type === 'click') setIsTooltipVisible((prev) => !prev);
  }
};
```